backend/main.py

```diff
@@ -4,11 +4,13 @@
 from fastapi import FastAPI
 from middleware.auth import AuthMiddleware
 from routers.routes.courses import course_router
+from routers.routes.posts import post_router
 from routers.routes.users import user_router
 
 app = FastAPI()
 app.include_router(user_router)
 app.include_router(course_router)
+app.include_router(post_router)
```

Comment:
- Pull from `dev` and see how the `courses.py` is setting `documents` as its subpath